packages/viewer/src/IViewer.ts

```diff
@@ -57,4 +58,9 @@ export interface IViewer {
   getObjectsProperties(includeAll?: boolean): unknown
 
   dispose(): void
+
+  thickLines: boolean
```

Comment:
All of these props were expected to be defined on IViewer, cause Sandbox.ts used them on the viewer typed as IViewer